packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -38,23 +39,26 @@ class SkiaGoldClient {
 
   /// The file system to use for storing the local clone of the repository.
   ///
-  /// This is useful in tests, where a local file system (the default) can
-  /// be replaced by a memory file system.
+  /// This is useful in tests, where a local file system (the default) can be
+  /// replaced by a memory file system.
   final FileSystem fs;
 
   /// A wrapper for the [dart:io.Platform] API.
   ///
-  /// This is useful in tests, where the system platform (the default) can
-  /// be replaced by a mock platform instance.
+  /// This is useful in tests, where the system platform (the default) can be
+  /// replaced by a mock platform instance.
   final Platform platform;
 
   /// A controller for launching sub-processes.
   ///
-  /// This is useful in tests, where the real process manager (the default)
-  /// can be replaced by a mock process manager that doesn't really create
+  /// This is useful in tests, where the real process manager (the default) can
+  /// be replaced by a mock process manager that doesn't really create
   /// sub-processes.
   final ProcessManager process;
 
+  /// What testing environment we may be in, like Cirrus or Luci.
+  final String ci;
```

Comment:
Maybe define an enum for the possible values that this can take? Passing and comparing strings seems error-prone.

Also use switch/case to check the enum value.

Reply:
Good call! Done. PTAL.